.evergreen/build-mac.sh

```diff
@@ -8,22 +8,24 @@ rm -rf validdist
 mkdir -p validdist
 mv dist/* validdist || true
 
-for VERSION in 3.7 3.8 3.9 3.10; do
-    PYTHON=/Library/Frameworks/Python.framework/Versions/$VERSION/bin/python3
-    rm -rf build
+VERSION=${VERSION:-3.10}
+PYTHON=/Library/Frameworks/Python.framework/Versions/$VERSION/bin/python3
+rm -rf build
 
-    createvirtualenv $PYTHON releasevenv
-    python -m pip install --upgrade wheel
-    python -m pip install setuptools==63.2.0
-    python setup.py bdist_wheel
-    deactivate || true
-    rm -rf releasevenv
+createvirtualenv $PYTHON releasevenv
+python -m pip install --upgrade wheel
+python -m pip install setuptools==63.2.0
+python setup.py bdist_wheel
+deactivate || true
+rm -rf releasevenv
 
-    # Test that each wheel is installable.
-    for release in dist/*; do
-        testinstall $PYTHON $release
-        mv $release validdist/
-    done
+# Test that each wheel is installable.
+for release in dist/*; do
+    if [[ "$release" =~ .*universal2.* ]]; then
+        export ENSURE_UNIVERSAL2="1"
```

Comment:
Per my last comment I think we should move ENSURE_UNIVERSAL2=1 to release-mac-1100, that way if the job stops producing universal2 wheels for some reason, we'll know:
```
    - name: "release-mac-1100"
      tags: ["release_tag"]
      run_on: macos-1100
      commands:
        - func: "build release"
          vars:
              VERSION: "3.10"
              ENSURE_UNIVERSAL2: "1"
        - func: "build release"
          vars:
              VERSION: "3.9"
              ENSURE_UNIVERSAL2: "1"
        - func: "upload release"
 ```

Reply:
Done